assets/src/edit-story/app/prepublish/error/metadata.js

```diff
@@ -170,33 +177,23 @@ export function publisherLogoSize(story) {
  * If there is an element with a link in the page attachment region, return an error message.
  * Otherwise, return undefined.
  *
- * @param {Story} story The story being checked for critical metadata
+ * @param {Page} page The story being checked for critical metadata
  * @return {Guidance|undefined} Guidance object for consumption
  */
-export function linkInPageAttachmentRegion(story) {
-  const { pages } = story;
-  const pagesWithLinksInAttachmentArea = pages
-    .filter((page) => {
-      const { elements } = page;
-      const hasPageAttachment = Boolean(page.pageAttachment?.url?.length);
-      return (
-        hasPageAttachment &&
-        elements
-          .filter(({ link }) => Boolean(link?.url?.length))
-          .some(isElementBelowLimit)
-      );
-    })
-    .map((page) => page.id);
-
-  const isLinkInPageAttachmentArea = Boolean(
-    pagesWithLinksInAttachmentArea.length
-  );
+export function linkInPageAttachmentRegion(page) {
```

Comment:
this change is linked to [this issue](https://github.com/google/web-stories-wp/issues/6081)